src/services/community-service.js

```diff
@@ -51,7 +50,7 @@ const updateCommunity = async (objectCommunity, communityId) => {
 const deleteCommunity = async (communityId) => {
     try {
         const data = await communityData.deleteCommunity(communityId);
-        if(data[0].affectedRows === 0){
+        if(data.affectedRows === 0){
```

Comment:
same comment as above the repository should handle the non existence of the entity